### PR TITLE
Temporarily remove some plugins where vsix files are missing

### PR DESCRIPTION
**Pull request by ericwill**
Signed-off-by: Eric Williams <ericwill@redhat.com>

### What does this PR do?
Temporarily remove plugins that have dead vsix links, until we can replace the links and enable offline builds by default.

### What issues does this PR fix or reference?

Part of eclipse/che#18933

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [x] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [x] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: che-theia-plugins.yaml [hunk 1]
```diff
@@ -135,20 +135,6 @@ plugins:
...
-    aliases:
-      - haskell/vscode-haskell
-    sidecar:
-      directory: haskell
-      name: vscode-haskell
-      memoryLimit: 3072Mi
-    extensions:
-      - https://open-vsx.org/api/haskell/haskell/1.1.0/file/haskell.haskell-1.1.0.vsix
-      - https://open-vsx.org/api/justusadam/language-haskell/3.3.0/file/justusadam.language-haskell-3.3.0.vsix
-      - https://open-vsx.org/api/gattytto/phoityne-vscode/0.0.28/file/gattytto.phoityne-vscode-0.0.28.vsix
```
Comment (sunix) [inline che-theia-plugins.yaml:151]: I am wondering if we could just disable `phoityne-vscode`. Not sure what is the impact.

**Comment by sunix** [comment]
Should be part of the next che hotfix z release https://github.com/eclipse/che/issues/18926 7.25.2

File: che-theia-plugins.yaml [hunk 2]
```diff
@@ -371,11 +357,6 @@ plugins:
     extensions:
       - https://download.jboss.org/jbosstools/vscode/3rdparty/vscode-kubernetes-tools/vscode-kubernetes-tools-1.2.1.vsix
       - https://download.jboss.org/jbosstools/vscode/3rdparty/vscode-yaml/vscode-yaml-0.8.0.vsix
-  - repository:
-      url: https://github.com/rogalmic/vscode-bash-debug
-      revision: v0.3.9
-    extensions:
-      - https://open-vsx.org/api/rogalmic/bash-debug/0.3.9/file/rogalmic.bash-debug-0.3.9.vsix
```
Comment (benoitf) [inline che-theia-plugins.yaml:378]: https://github.com/rogalmic/vscode-bash-debug/releases/download/untagged-78eaa7784342409ec2a3/bash-debug-0.3.9.vsix ?

**Review by benoitf — APPROVED** [review]

File: che-theia-plugins.yaml (hunk 1 above)
Comment (ericwill) [reply che-theia-plugins.yaml:151]: I didn't contribute this plugin so I don't know. Better remove it and we can add it back later.

File: che-theia-plugins.yaml (hunk 2 above)
Comment (ericwill) [reply che-theia-plugins.yaml:378]: I'll re-introduce it in a separate PR

Comment (ericwill) [reply che-theia-plugins.yaml:378]: Done :+1:

**Pull request merged**